### PR TITLE
feat: split payment preview skeleton loader

### PR DESCRIPTION
**Pull request by joanna-pagepro**
## Description

Split payment preview loading state

## Testing

* Create a split payment.
* Select the Reputation percentage distribution type.
* Select users with reputation.
* Create the action.
* Note the loading state of the split payment table.

## Diffs

**New stuff** ✨

* 

**Changes** 🏗

* loading states added into missing places and members loading state added to the table to show the loader when members data is fetched

**Deletions** ⚰️

* 

## TODO

- 

Resolves https://github.com/JoinColony/colonyCDapp/issues/2843

### CODE REVIEW

File: src/components/v5/common/CompletedAction/partials/SplitPayment/partials/SplitPaymentTable/hooks.tsx [hunk 1]
```diff
@@ -59,18 +62,32 @@ export const useGetSplitPaymentColumns = (
...
           const token = getSelectedToken(colony, data[0]?.tokenAddress);
 
-          return token ? (
-            <div className="flex items-center justify-end gap-9 md:justify-start">
-              <SplitPaymentPayoutsTotal data={data} token={token} />
-              {isTablet && (
-                <span className="text-md font-medium text-gray-900">100%</span>
+          return (
+            <>
+              {isLoading ? (
```
Comment (rumzledz) [inline src/components/v5/common/CompletedAction/partials/SplitPayment/partials/SplitPaymentTable/hooks.tsx:67]: You can use the `LoadingSkeleton` 💀  component here:

```js
// You can also pass `classNames` to use a specific border radius/width/height etc. :)
<LoadingSkeleton isLoading={isLoading}>
  {token ? (
    <div className="flex items-center justify-end gap-9 md:justify-start">
      <SplitPaymentPayoutsTotal data={data} token={token} />
      {isTablet && (
        <span className="text-md font-medium text-gray-900">100%</span>
      )}
    </div>
  ) : null}
</LoadingSkeleton>
```

File: src/components/v5/common/CompletedAction/partials/SplitPayment/partials/SplitPaymentTable/hooks.tsx [hunk 2]
```diff
@@ -86,7 +103,7 @@ export const useGetSplitPaymentColumns = (
             ? calculatePercentageValue(data?.[row.index].amount, amount)
             : 0;
 
-          return isLoading ? (
+          return isDataLoading ? (
```
Comment (rumzledz) [inline src/components/v5/common/CompletedAction/partials/SplitPayment/partials/SplitPaymentTable/hooks.tsx:106]: Here as well 💀

File: src/components/v5/common/CompletedAction/partials/SplitPayment/partials/SplitPaymentTable/hooks.tsx [hunk 3]
```diff
@@ -97,16 +114,22 @@ export const useGetSplitPaymentColumns = (
           );
         },
         footer: !isTablet
-          ? () => (
-              <span className="text-md font-medium text-gray-900">100%</span>
-            )
+          ? () =>
+              isDataLoading ? (
+                <div className="flex w-[4rem] items-center">
```
Comment (rumzledz) [inline src/components/v5/common/CompletedAction/partials/SplitPayment/partials/SplitPaymentTable/hooks.tsx:119]: And here 💀

**Review by rumzledz — CHANGES_REQUESTED** [review]
Nice and straightforward @joanna-pagepro ! ✨ I did leave a tiny change request regarding the `LoadingSkeleton` component but that's pretty much it 👌

**Review by mmioana — APPROVED** [review]
Nice work @joanna-pagepro 🌟 

Followed your testing steps and can confirm the loading skeletons are showing up

https://github.com/user-attachments/assets/c58b37db-555b-41a0-b28a-985e90ea3635

Besides the `LoadingSkeleton` usage as @rumzledz proposed, everything looks good from my side

**Comment by joanna-pagepro** [comment]
@rumzledz @mmioana updated

**Review by rumzledz — APPROVED** [review]
The table's loaders are looking 💯 

loaders split

**Pull request merged**